vortex-serde/src/layouts/write/writer.rs

```diff
@@ -222,4 +230,16 @@ mod tests {
         let written = block_on(async { writer.finalize().await }).unwrap();
         assert!(!written.is_empty());
     }
+
+    #[test]
+    fn postscript_size() {
+        let ps = Postscript::new(1000000u64, 1100000u64);
+        let mut fbb = FlatBufferBuilder::new();
+        let ps_fb = ps.write_flatbuffer(&mut fbb);
+        fbb.finish_minimal(ps_fb);
+        let (buffer, buffer_begin) = fbb.collapse();
+        let buffer_end = buffer.len();
+
+        assert_eq!(buffer[buffer_begin..buffer_end].len(), 32);
```

Comment:
check this against the constant?

Reply:
Yes, I realized I made this test but didn't connect it to the constant

Reply:
should also check that it's exactly 32, since we'll want to increment the version if it changes